e2e/src/main/java/gov/hhs/cdc/trustedintermediary/e2e/SentPayloadReader.java

```diff
@@ -0,0 +1,28 @@
+package gov.hhs.cdc.trustedintermediary.e2e;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class SentPayloadReader {
+
+    public static String read() throws IOException {
+        Path payloadFile = findFilePayload();
+
+        return Files.readString(payloadFile);
+    }
+
+    public static Path findFilePayload() {
```

Comment:
```suggestion
    private static Path findFilePayload() {
```
This can be a private method since nothing outside needs to call it.